enigma-principal/enclave/Enclave.config.xml

```diff
@@ -4,7 +4,7 @@
   <ISVSVN>0</ISVSVN>
   <StackMaxSize>0x40000</StackMaxSize>
   <HeapMaxSize>0x100000</HeapMaxSize>
-  <TCSNum>1</TCSNum>
+  <TCSNum>10</TCSNum>
```

Comment:
I'm not sure if this is a good idea.
Do you know what does this do? 
do you know if there's any consequences for this?

Reply:
Found something here:
https://download.01.org/intel-sgx/linux-2.5/docs/Intel_SGX_Developer_Reference_Linux_2.5_Open_Source.pdf
page 65. 

```
TCSMaxNum, TCSNum, and TCSMinPool are used to determine how many
threads will be created after the enclave initialization, and how many threads
can be created dynamically when the enclave is running. These two kinds of
threads are referred to as static threads and dynamic threads respectively.
```


Do you launch threads?